poetry.lock

```diff
@@ -119,64 +119,61 @@ pyflakes = ">=2.5.0,<2.6.0"
 
 [[package]]
 name = "grpcio"
-version = "1.50.0"
+version = "1.51.1"
```

Comment:
Is regenerating the lockfile necessary?

Reply:
My local setup complained it was out of sync